scripts/build-color-variants.js

```diff
@@ -300,51 +281,59 @@ function buildColorVariants(variables, config) {
 
         input:checked + .toggle--${color} {
           background: var(--${color});
-          color: var(--white);
         }
+      `);
+    }, '');
+  };
 
+  variantGenerators.toggleActive = function (colors) {
+    return colors.reduce((result, color) => {
+      if (isNotSuitableForForms(color)) return result;
+      // Must be below .toggle group in stylesheet
+      return result += stripIndent(`
         input:checked + .toggle--active-${color} {
           color: var(--${color});
         }
       `);
     }, '');
   };
 
-  variantGenerators.toggleActive = function (colors) {
+  variantGenerators.switch = function (colors) {
     return colors.reduce((result, color) => {
-      // Must be below .toggle group in stylesheet
+      if (isNotSuitableForForms(color)) return result;
+      const darkerShade = getDarkerShade(color);
+      // Darken background when hovered and when active
+      // Darken dot on hover when inactive only
       return result += stripIndent(`
-        input:checked + .toggle--active-${color} {
+        .switch--${color} {
           color: var(--${color});
         }
+
+        .switch--${color}:hover {
+          color: var(--${darkerShade});
+        }
+
+        .switch--${color}:hover::after,
+        input:checked + .switch--${color} {
+          background-color: var(--${darkerShade});
+        }
+
+        input:checked + .switch--dot-${color}::after {
+          background-color: var(--${color});
+        }
       `);
     }, '');
   };
 
   variantGenerators.range = function (colors) {
```

Comment:
Good catch - was from an earlier refactor attempt. I removed this class but didn't remove the documentation?